linode/accountavailability/framework_datasource_schema.go

```diff
@@ -0,0 +1,22 @@
+package accountavailability
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var frameworkDataSourceSchema = schema.Schema{
+	Attributes: Attributes,
+}
+
+var Attributes = map[string]schema.Attribute{
+	"region": schema.StringAttribute{
+		Description: "The id of a region.",
+		Required:    true,
+	},
+	"unavailable": schema.ListAttribute{
```

Comment:
It might make sense to make this a `SetAttribute` because the API does not guarantee the order of this list 👍